### ISSUE TITLE
Input queue is empty when calling next_batch

### ISSUE THREAD

**Issue by chmille3** [abisee/pointer-generator#114]
I am using this command to run:

python run_summarization.py --mode=decode --data_path=./finished_files/chunked/test_* --vocab_path=./finished_files/vocab --log_root=./pretrained_model --max_enc_steps=400 --max_dec_steps=120 --coverage=1

But I am getting this warning and nothing happens:
WARNING:tensorflow:Bucket input queue is empty when calling next_batch. Bucket queue size: 0, Input queue size: 400

Any help is appreciated, thanks!

**Comment by chmille3** [comment]
I take out the --max_enc_steps=400 and I get this error:

INFO:tensorflow:Failed to load checkpoint from ./pretrained_model/train. Sleeping for 10 secs...

**Comment by Zerone0101** [comment]
> I am using this command to run:
> 
> python run_summarization.py --mode=decode --data_path=./finished_files/chunked/test_* --vocab_path=./finished_files/vocab --log_root=./pretrained_model --max_enc_steps=400 --max_dec_steps=120 --coverage=1
> 
> But I am getting this warning and nothing happens:
> WARNING:tensorflow:Bucket input queue is empty when calling next_batch. Bucket queue size: 0, Input queue size: 400
> 
> Any help is appreciated, thanks!

did you find the cause of this problem, i met this problem too.

**Comment by chmille3** [comment]
It has been awhile since I have worked on this, but I believe the source of the problem was the version of tensorflow I was using with respect to the pre-trained model. I used the pre-trained model for tensorflow 1.2.1. I got mine to work by used that version of tensorflow installed. I hope this helps.

**Comment by TanyaChowdhury** [comment]
Was anyone able to figure out why this is happening?
Changing the tf version didn't work for me.

**Comment by ghost** [comment]
Have you solved it now? Changing the TF version is useless for me.

**Comment by TanyaChowdhury** [comment]
Yes, I think it was about getting to the exact TF versions for me.
